assets/scss/modules/_forms.scss

```diff
@@ -206,11 +202,17 @@
   }
 
   input[type='checkbox'] {
-    opacity: 0;
-    z-index: 0;
     cursor: pointer;
-    zoom: 2.5;
     position: absolute;
+    width: 14px;
+    height: 14px;
+
+    &:focus {
+      outline: 4px solid #e79433;
```

Comment:
I understand that the solution for the _checkbox_ is different than in other _inputs_, but, what do you think about to keep it closer? I mean, in place of declare the style here, do it near the other focus declarations:

```sass
  select:focus,
  textarea:focus,
  input:focus {
    outline: none;
    border: 1px solid #e79433;
    box-shadow: inset 0 0 0 2px #e79433;
  }

   input[type='checkbox']:focus {
      outline: 4px solid #e79433;
      outline-offset: 0px;
      top: 1px;
      left: 1px;
    }
```